CHANGELOG-4.0.md

```diff
@@ -6,14 +6,16 @@ in 4.0 versions.
 
 * 4.0.0 (xxxx-xx-xx)
 
- * Add `ruflin/Elastica` 3.x support.
```

Comment:
ups, you are right, i got confused between the 3.2.x branch and master. if people relied on dev-master or 4.0 @ dev then making that suddenly be for ES 5, it could create confusion. maybe we could create a tag as es-3-best-effort or something like that, so that at least that state can be references if somebody needs it.